source/blender/gpu/intern/gpu_texture.c

```diff
@@ -726,6 +726,11 @@ void GPU_texture_ref(GPUTexture *tex)
 	tex->refcount++;
 }
 
+int GPU_texture_ref_count(GPUTexture *tex)
+{
+	return tex->refcount;
+}
```

Comment:
Isn't it easier to just call tex->refcount instead of an entire new method? :question: